smdebug/profiler/trace_event_file_parser.py

```diff
@@ -0,0 +1,128 @@
+# First Party
+from smdebug.core.logger import get_logger
+
+TIMESCALE_MULTIPLIER = {"ns": 1000, "us": 1, "ms": 0.001, "s": 0.000001}
```

Comment:
if tensorflow says displayTimeUnit="ns" , does the time reported there are in ns or some other unit? 

I was under impression that displayTimeUnit tells what unit is for time in trace.json file. Individual reader can have its own _timescale_multiplier_for_ns which defines what to multiply to convert into ns time format. Example SMTFProfiler knows events are in us, so multiplier would be 1000 to convert to ns. 
NS is time interval that reader normalizes all events to. This normalization will give us unique timeline view if there are timeline files from multiple sources. 

Thinking this I kept 
{"ns": 1, "us": 1000, "ms": 1000000, "s": 1000000000}

That means that if displayTimeUnit is "us", we will multiply by 1000 and so on. 
Can you decribe and write mroe comments about displayTimeUnit and timescale multiplier.

Reply:
@Vikas-kum  As per the Trace event format https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/edit  The 'ts' is always in uSecond granularity

"ts: The tracing clock timestamp of the event. The timestamps are provided at microsecond granularity.
"

In case of Tensorboard generated trace file, the 'displayTimeUnit' is specified to be 'ns'. This is a hint to display that for higher precision show the events on 'ns' scale. Therefore, we would have to multiply it by 1000, if we want to keep the same precision.  

The short answer, as per the standard, the 'ts' and 'durations' will always be in uSeconds. 
It will be up to us whether we want to convert into ns scale for precise calculations. IMO, we should always convert it to ns scale.

Reply:
yeah in that case we should ignore displayTimeUnit, and always convert from us and default TIMESCLAE_MULTIPLIER to 1000 .

Reply:
> yeah in that case we should ignore displayTimeUnit, and always convert from us and default TIMESCLAE_MULTIPLIER to 1000 .

@Vikas-kum 
Ok sounds good. I will remove the code for reading the displayTimeUnit and will always convert the timestamp values (the 'ts' field and start_time_since_epoch_in_micros) from microSeconds to nanoSeconds.
I will also cut down the sample json files for testing.